cpp/src/jit/types_h_jit.cpp

```diff
@@ -0,0 +1,5 @@
+// From Nik's answer in
+// https://stackoverflow.com/questions/10422034/when-to-use-extern-in-c/10422050#10422050
+// In C++ the extern specifier has to be applied to the variable definition as well.
+extern 
```

Comment:
Are you missing a variable name here? There's an `extern` with nothing coming after it.

Reply:
I'm confused about what this file is for. It's an empty `.cpp` that just includes another header.

Reply:
`types.h.jit` is a header that contains a string which contains the content of `cudf/types.h` wrapped up in a `const` string called `cudf_types_h` using the `stringify` binary produced in `jitify`. `types.h.jit` is placed under `build/include`. This string is now used in more than one TU so we need the `extern` here. This is used by `jitify`. This does seems ugly but I can't think of a better way to do this other than modifying `stringify` itself. Maybe @devavret can think of a better way to do this?